opacclient/libopac/src/main/java/de/geeksfactory/opacclient/objects/AccountItem.java

```diff
@@ -0,0 +1,148 @@
+package de.geeksfactory.opacclient.objects;
+
+public abstract class AccountItem {
+    private long account;
+    private String title;
+    private String author;
+    private String format;
```

Comment:
I didn't do anything about this yet. Should I still add the `SearchResult.MediaType` property?